crates/ratchet-web/src/model.rs

```diff
@@ -203,6 +203,43 @@ impl Model {
         })
     }
 
+    #[wasm_bindgen]
+    pub async fn load_custom(
```

Comment:
This method and load are 99% the same.

Can't we extract the core logic into `load_inner`, and then expose `load_custom` and `load`?

Reply:
Will do

Reply:
Nice!